WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+SectionHelpers.swift

```diff
@@ -60,4 +60,28 @@ extension BlogDetailsViewController {
         }
         return blog.supports(.themeBrowsing) || blog.supports(.menus)
     }
+
+    @objc func shouldAddSharingRow() -> Bool {
+        guard JetpackFeaturesRemovalCoordinator.jetpackFeaturesEnabled() else {
+            return false
+        }
+        return blog.supports(.sharing)
+    }
+
+    @objc func shouldAddPeopleRow() -> Bool {
+        guard JetpackFeaturesRemovalCoordinator.jetpackFeaturesEnabled() else {
+            return false
+        }
+        return blog.supports(.people)
+    }
+
+    @objc func shouldAddPluginsRow() -> Bool {
+        return blog.supports(.pluginManagement)
+    }
+
+    @objc func shouldAddDomainRegistrationRow() -> Bool {
```

Comment:
As I understand, we'll continue supporting domain registration for the WP app?

Reply:
Okay, I see, it's controlled by `AppConfiguration.allowsDomainRegistration` rather than by `jetpackFeaturesEnabled() `.  👍

In the ideal scenario in the future, we may want to have one mechanism to determine the differences between WP and JP.

Reply:
> In the ideal scenario in the future, we may want to have one mechanism to determine the differences between WP and JP.

I agree!